packages/web-app-files/src/components/SideBar/FileInfo.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="file_info oc-flex oc-flex-between">
+  <div class="file_info oc-flex oc-flex-between oc-px-s oc-py-m">
```

Comment:
```suggestion
  <div class="file_info oc-flex oc-flex-between oc-p-s">
```
Vertical `m` size padding feels too big for me. I'd like to switch to `s` size padding for all directions.

Reply:
I was only talking about the vertical alignment = lowering the horizontal padding. Benedikt suggests lowering the vertical padding as well, if I'm not mistaken 🙂

Reply:
Yes, exactly